shared/serverlist_state.js

```diff
@@ -4,35 +4,33 @@ let lastChecked = Date.now()
 const { GameServer, GetGameServers, RemoveGameServer } = require( "../shared/gameserver.js" )
 
 module.exports = {
-	getLastChecked()
-	{
+	getLastChecked: function () {
```

Comment:
These are identical in ES2015.

Reply:
Better fits formatting of the rest of master server code